app/javascript/src/components/Common/Header.jsx

```diff
@@ -3,13 +3,13 @@ import React, { useState } from "react";
 import { Button } from "neetoui";
 import { Header as NeetoHeader } from "neetoui/layouts";
 
-const Header = ({ title, setShowMenu, setShowPane }) => {
+const Header = ({ title, setShowMenu, setShowPane, showMenu }) => {
   const [searchTerm, setSearchTerm] = useState("");
 
   return (
     <NeetoHeader
       title={`All ${title}`}
-      menuBarToggle={() => setShowMenu(prev => !prev)}
+      menuBarToggle={() => setShowMenu(!showMenu)}
```

Comment:
Not sure why this change was made over here. What's your strong reasoning for doing this?

Reply:
It was a mistake from my side. Resolved in #14